### PR TITLE
W-13678690 save excel file cv

### PR DESCRIPTION
**Pull request by Cristian-Venticinque**

### CODE REVIEW

File: rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations.adoc [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
 * xref:toolbox-excel-operations-recalculate-excel-file.adoc[Recalculate Excel File]
 +
-This Action Step enables you to recalculate an Excel sheet
+Use Recalculate Excel File to recalculate the values of an Excel sheet.
+* xref:toolbox-excel-operations-save-excel-file.adoc[Save Excel File]
++
+Use Save Excel File to save the contents of an Excel file.
 * xref:toolbox-excel-operations-write-to-excel-file.adoc[Write to Excel File]
++
+Use Write to Excel File to write the contents from the read Excel file. You can write the content to the same file or to a new destination file.
```
Comment (ms-jscholten) [inline rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations.adoc]: **suggestion**: Write to Excel file can be used to write any content to the excel file. The text reads as if only content that was read from the Read Excel File action step can be used.

```suggestion
Use Write to Excel File to write content to the selected excel file. You can write the content to the same file or to a new destination file.
```

File: rpa-builder/6.4/modules/ROOT/nav.adoc [hunk 2]
```diff
@@ -43,6 +43,7 @@
 *** xref:toolbox-excel-operations-excel-session.adoc[Excel Session]
 *** xref:toolbox-excel-operations-read-from-excel-file.adoc[Read from Excel File]
 *** xref:toolbox-excel-operations-recalculate-excel-file.adoc[Recalculate Excel File]
+*** xref:toolbox-excel-operations-save-excel-file.adoc[]
```
Comment (ms-jscholten) [inline rpa-builder/6.4/modules/ROOT/nav.adoc:46]: **issue**: Looks like the title for the nav is missing inside the [ ]

File: rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-save-excel-file.adoc [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Use the Save Excel File Action Step to save the changes to the Excel file being modified inside the Excel Session at any point of the execution. For example, saving the Excel file after loop operations serves as a way to prevent data loss in case the workflow execution stops because of an unknown error. The Excel Session automatically saves the file at the end of its execution. 
+
+The Save Excel File Action Step works only inside an Excel Session. Specify the file to modify in the xref:toolbox-excel-operations-excel-session.adoc[Excel Session].
+
+[WARNING]
+Save Excel File cannot save the file if the Excel Session opens the file in read-only mode.
+
+== Properties
+
+This Action Step does not have any specific properties. See xref:common-properties.adoc[] for a list of properties common to all Action Steps.
```
Comment (ms-jscholten) [inline rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-save-excel-file.adoc:12]: **issue**: Looks like the title for the nav is missing inside the [ ]

File: rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-save-excel-file.adoc [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+== Inbound Variables
+
+Save Excel File has no inbound variables.
+
+== Outbound Variables
+
+* *Success* (bool)
++
+Confirms if the save file operation was successfull. If the operation fails (`false`), the workflow run state is set to Error.
```
Comment (ms-jscholten) [inline rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-save-excel-file.adoc]:
```suggestion
Confirms if the save file operation was successful. If the operation fails (`false`), the workflow run state is set to Error. 

```

File: rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-save-excel-file.adoc (hunk 3 above)
Comment (Cristian-Venticinque) [reply rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-save-excel-file.adoc:12]: When it is empty, it auto-populates with the Title of the article

File: rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations.adoc [hunk 5]
```diff
@@ -1,19 +1,23 @@
...
 * xref:toolbox-excel-operations-recalculate-excel-file.adoc[Recalculate Excel File]
 +
-This Action Step enables you to recalculate an Excel sheet
+Use Recalculate Excel File to recalculate the values of an Excel sheet.
+* xref:toolbox-excel-operations-save-excel-file.adoc[Save Excel File]
++
+Use Save Excel File to save the contents of an Excel file.
 * xref:toolbox-excel-operations-write-to-excel-file.adoc[Write to Excel File]
++
+Use Write to Excel File to write content to the selected excel file. You can write the content to the same file or to a new destination file.
```
Comment (Cristian-Venticinque) [inline rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations.adoc]:
```suggestion
Use Write to Excel File to write content to the selected Excel file. You can write the content to the same file or to a new destination file.
```

**Review by JennyHajee — APPROVED** [review]

**Pull request merged**